src/ui/components/confirm_transaction.h

```diff
@@ -43,4 +43,13 @@ component_t* confirm_transaction_fee_create(
     void (*confirm_callback)(void),
     void (*cancel_callback)(void));
 
+/**
```

Comment:
docstring incomplete, describe all params and the expected behavior of the component.